SentinelHub/tests/test_utils_geo.py

```diff
@@ -0,0 +1,69 @@
+# coding=utf-8
+from typing import Tuple, TypeVar
+
+import pytest
+
+pytest.importorskip("qgis.core")
+
+from qgis.core import QgsRectangle  # noqa: E402
+
+from ..settings import Settings  # noqa: E402
+from ..utils.geo import bbox_to_string, get_custom_bbox, is_bbox_too_large, is_supported_crs  # noqa: E402
+
+T = TypeVar("T", float, int)
```

Comment:
Is this meant to be a proper type variable, or is it just a replacement for `Union[float, int]`? Keep in mind that according to [PEP484](https://peps.python.org/pep-0484/#the-numeric-tower) if you have a type that can be either int or float, the annotation of `float` suffices